s/features/notes/components/notes/xiome-notes.ts

```diff
@@ -1,156 +1,156 @@
-
-import styles from "./xiome-notes.css.js"
-import {makeNotesModel} from "../../models/notes-model.js"
-import {renderOp} from "../../../../framework/op-rendering/render-op.js"
-import {ComponentWithShare, mixinStyles, html} from "../../../../framework/component.js"
-import {ops} from "../../../../framework/ops.js"
+import styles from "./xiome-notes.css.js";
```

Comment:
something has gone wrong in this file. i think you've accidentally run some kind of auto formatter.

- the indentation has all changed to spaces (we use tabs)
- semicolons were added (we don't use semicolons)
- etc

we should revert this change